hack/scripts/run-non-olm-kuttl-test.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+export NON_OLM="true"
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+sequential_suite=$ROOT/../../test/openshift/e2e/sequential/
+parallel_suite=$ROOT/../../test/openshift/e2e/parallel/
+
+testsuite="sequential"
+
+# Get test suite argument
+while getopts ":t:" opt; do
+  case ${opt} in
+    t) testsuite=$OPTARG;;
+    \?) echo "Please provide options sequential or parallel -$OPTARG" >&2;;
```

Comment:
I thought we agreed on executing all test suites if no option was specified. This looks like it will only execute `sequential` by default.